### ISSUE TITLE
Modifiers

### ISSUE THREAD

**Issue by andy-ms** [JamesHenry/tsep-babylon-test#6]
### Code

```ts
declare const enum E {}
```

### Eslint

```json
"modifiers": [
  {
    "type": "TSDeclareKeyword",
  }
  {
    "type": "TSConstKeyword",
  }
],
```

### Babylon

```json
"const": true,
"declare": true,
```

Using an array of modifiers resembles the way TypeScript handles it.
On the other hand, estree typically uses booleans for modifiers, so we won't have things like `export` or `static` in the modifiers array.

**Comment by JamesHenry** [comment]
I agree - I have suggested that we move to a more boolean flag orientated approach in the past with `declare` in particular.

I will provisionally mark this as an ESLint change
